webpack/assets/javascripts/react_app/components/common/DateTimePicker/date-time-picker.scss

```diff
@@ -1,4 +1,4 @@
-.bootstrap-datetimepicker-widget.dropdown-menu.top {
+.bootstrap-datetimepicker-widget.dropdown-menu {
```

Comment:
Please change this into:
```
.bootstrap-datetimepicker-widget.dropdown-menu.top {
  margin-top: -10px;
}

.bootstrap-datetimepicker-widget.dropdown-menu.bottom {
  margin-top: 10px !important;
}
.bootstrap-datetimepicker-widget.dropdown-menu {
  max-width: none;

```
otherwise the popover arrow is inside the input instead of being under it

Reply:
Please see #6761, this is PR for stable branch, we need to do it first to develop.